lock-api/src/main/java/com/palantir/lock/HeldLocksGrant.java

```diff
@@ -106,7 +106,8 @@ public long getExpirationDateMs() {
      * Returns the set of locks which were successfully acquired, as a mapping
      * from descriptor to lock mode.
      */
-    public SortedLockCollection<LockDescriptor> getLocks() {
+    @Override
+    public SortedLockCollection<LockDescriptor> getLockDescriptors() {
```

Comment:
This looks like an API break. I can see why you're doing it, but to avoid the break, it would be polite to have a `@Deprecated getLocks()` method that delegates to `getLockDescriptors()`.